frontend/src/@components/coupon/CouponCreateForm/index.tsx

```diff
@@ -80,25 +83,21 @@ const CouponCreateForm = (props: CouponCreateFormProps) => {
       )}
 
       <SelectInput label='어떤 쿠폰인가요 ?'>
-        {couponTypeCollection.map(({ engType }) => (
-          <Styled.TypeOption
-            key={engType}
-            isSelected={engType === currentCouponType}
-            onClick={onSelectCouponType(engType)}
-          >
-            <img src={THUMBNAIL[engType]} alt='쿠폰 종류' width={50} height={50} />
+        {couponTypeCollection.map(({ engType, koreanType }) => (
+          <Styled.TypeOption key={engType} isSelected={engType === currentCouponType}>
+            <button type='button' onClick={onSelectCouponType(engType)}>
+              <img src={THUMBNAIL[engType]} alt={koreanType} width={50} height={50} />
+            </button>
           </Styled.TypeOption>
         ))}
       </SelectInput>
 
       <SelectInput.VerticalView label='당신의 마음을 선택해주세요 !'>
         {couponHashtags.map(hashtag => (
-          <Styled.FeelOption
-            key={hashtag}
-            isSelected={hashtag === currentCouponTag}
-            onClick={onSelectCouponTag(hashtag)}
-          >
-            #{hashtag}
+          <Styled.FeelOption key={hashtag} isSelected={hashtag === currentCouponTag}>
+            <button type='button' onClick={onSelectCouponTag(hashtag)}>
+              #{hashtag}
+            </button>
           </Styled.FeelOption>
```

Comment:
voiceover가 #을 "숫자기호"라고 읽던데요! 저는 아래와 같이 aria-label을 달아주는건 보았는데, 어때요?

```suggestion
            <button type='button' aria-label={`hashtag ${couponTag}` onClick={onSelectCouponTag(hashtag)}>
              #{hashtag}
            </button>
          </Styled.FeelOption>
```